app/feed-event-service/feed-event.mocks.ts

```diff
@@ -0,0 +1,23 @@
+import {FeedEvent} from "../feed-event/feed-event";
+import {DOSE_AMIGOS_USERS} from "../dose-amigos-user-service/dose-amigos-user-mocks";
+
+export var FEED_EVENTS: Array<FeedEvent> = [
+    {
+        id: 3,
+        actionDateTime: DOSE_AMIGOS_USERS[0].lastTimeDoseTaken,
+        user: DOSE_AMIGOS_USERS[0],
+        action: "has completed their medication."
+    },
+    {
+        id: 2,
+        actionDateTime: DOSE_AMIGOS_USERS[1].lastTimeDoseTaken,
+        user: DOSE_AMIGOS_USERS[1],
+        action: "has completed their medication."
+    },
+    {
+        id: 1,
+        actionDateTime: DOSE_AMIGOS_USERS[2].lastTimeDoseTaken,
+        user: DOSE_AMIGOS_USERS[2],
+        action: "has completed their medication."
```

Comment:
I think we group by scheduleTime and sort by actionTime DESC ?

Since we only create a FeedEvent when the last DoseEvent gets completed, then that is the actionTime, right? But all of the 8:00AM medicine has the same scheduleTime.  However we want to represent that.

FeedEvent needs a DoseAmigosUser, an actionType (Taken, missed, skipped), when it was scheduled (so we can say "taken all 8:00 AM meds"), and the actionTime.  Do we want to have it group DoseEvents in a List, or just flatten it out?

Either way, we can handle in a follow up ticket, I'm merging this one that just renames the object.